### ISSUE TITLE
Spacy: Language not suporte

### ISSUE THREAD

**Issue by Ueland** [alephdata/aleph#427]
Hi!

Currently doing some testing of Aleph against Norwegian documents, i notice the following in the log upon indexing documents with Norwegian content. Allthough it is considered a DEBUG notice, i am still inerested in what it results it. What kind of (meta)data extraction  failes due to missing language support?

And how much work will it be to add support for more languages?

Feel free to add it to the FAQ if it is a trivial matter 

`
extract-spacy_1     | DEBUG:service:Language not suported: no
extract-spacy_1     | DEBUG:service:Language not suported: no
extract-spacy_1     | DEBUG:service:Language not suported: no
extract-spacy_1     | DEBUG:service:Language not suported: no
extract-spacy_1     | DEBUG:service:Language not suported: no
extract-spacy_1     | DEBUG:service:Language not suported: no
extract-spacy_1     | DEBUG:service:Language not suported: no
extract-spacy_1     | DEBUG:service:Language not suported: no
`

**Comment by pudo** [comment]
Well, it has to do with Spacy itself: they only ship 7 language models, none of which is Norwegian: https://spacy.io/models/. That's why we have two NER services: polyglot is really good at languages (I think it has 38 models?), but generates slightly lower-quality taggings. Spacy has better quality but much fewer models.

**Comment by Ueland** [comment]
Yeah, i notice that polyglot does indeed support Norwegian so i guess this is a non-issue then :+1:
